bin/rialto/runtime/Cargo.toml

```diff
@@ -93,9 +93,9 @@ std = [
 	"pallet-aura/std",
 	"pallet-balances/std",
 	"pallet-bridge-eth-poa/std",
-	"pallet-bridge-call-dispatch/std",
+	"pallet-bridge-dispatch/std",
```

Comment:
The rename affects sorting, I think it would be good to fix in all `Cargo.toml` files (unfortunately fmt is not able to handle that afaik).

Reply:
I fixed the ones touched in this PR. It was actually pretty easy with [Vim's `sort` utility](https://vim.fandom.com/wiki/Sort_lines).

You can select the range you want (either with numbers or in visual mode) and then call `:sort`.